src/tlo/methods/cardio_metabolic_disorders.py

```diff
@@ -1556,6 +1566,8 @@ def apply(self, person_id, squeeze_factor):
 
         # Don't advise those with CKD to lose weight, but do so for all other conditions if BMI is higher than normal
         if self.condition != 'chronic_kidney_disease' and (df.at[person_id, 'li_bmi'] > 2):
+            self.EQUIPMENT.update({'Weighing scale'})
+
```

Comment:
Would the person be weighted every time they come for the same appointment in a few days because the medication was not available?

Reply:
okay